lib/turn.rb

```diff
@@ -0,0 +1,58 @@
+class Turn
+
+  attr_reader :computer, :user
+  def initialize(computer, user)
+    @computer = computer
+    @user = user
+  end
+
+  def display_boards
+    puts "=============COMPUTER BOARD============="
+    puts computer.render
+    puts "==============PLAYER BOARD=============="
+    puts user.render(true)
+  end
+
+  # player takes shot
+  # puts intro statement
+  # gets.chomp shot
+  # validate_shot(board, coordinate)
+
+  def user_shoots(computer)
+    puts "Enter coordinate for your shot: "
+    p "> "
```

Comment:
Let's edit this  `p ">"` so that it's not on a separate line from the user input

Reply:
Yes, love this. n